internal/wire/message.go

```diff
@@ -186,6 +187,9 @@ func makeEmptyMessage(command string) (Message, error) {
 	case CmdProtoconf:
 		msg = &MsgProtoconf{}
 
+	case CmdAuthch:
```

Comment:
Maybe
```golang
case CmdProtoconf, CmdAuthch:
    msg = &MsgProtoconf{}
```

Reply:
But it should not be the MsgProtoconf message. It should be a MsgAuthch?

Reply:
Yes, it's too dirty fix, I beleive we should implement "empty" handler for authch command

Reply:
Since we probably will change entire code responsible for peer communication it can stay as it is